README.md

```diff
@@ -8,10 +8,16 @@ MobilePay Online is essentially a way for the user to accept online payments in
 
 As a rule of thumb, the APIs are RESTful. You can expect POSTs to return the id of the resource created.
 
+## Sandbox environment
+
+The MobilePay Sandbox is a self-contained, testing environment that mimics the live MobilePay production environment. It provides you the space to play around and test your implementation and the requests you make to the MobilePay APIs without affecting MobilePay for the users.
+Link: https://sandbox-developer.mobilepay.dk/
+
```

Comment:
I think this part is good.
However, in the below, you've made some formatting in another PR that's working better